readme.md

```diff
@@ -104,6 +104,7 @@ Click the type names for complete docs.
 - [`Entries`](source/entries.d.ts) - Create a type that represents the type of the entries of a collection.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
+- [`Passthrough`](source/passthrough.d.ts) - A passthrough type that allows some types to be used to extend an interface, which originally can't.
```

Comment:
`A passthrough` is moot as it's just repeating the type name. Try to make the description follow the format of the other descriptions.

Reply:
Is the new one better?